packages/input-output-logger/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (opts) => {
+  const defaults = {
+    logger: console.log
+  }
+
+  const options = Object.assign({}, defaults, opts)
+
+  return ({
+    before: (handler, next) => {
+      if (typeof options.logger === 'function') {
+        options.logger(handler.event)
```

Comment:
I wonder if we should make clear that this log line concerns the event, by logging something like:

```javascript
options.logger('event', handler.event)
```

or 

```javascript
options.logger({ event: handler.event })
```

What do you guys think?

CC: @dkatavic @vladgolubev @nnyegaard @ossareh  ?

Reply:
I prefer option 1, but really with type destructuring in function arguments being so normal now I don't think it really matters either way?